mobile/src/main/res/values/strings.xml

```diff
@@ -153,11 +154,19 @@
     <!-- About -->
     <string name="about_title">About</string>
     <string name="about_license_title">License</string>
-    <string name="about_license" translatable="false"><![CDATA[<a href="https://raw.githubusercontent.com/openhab/openhab.android/master/LICENSE">Eclipse Public License - v 1.0</a>]]></string>
-    <string name="about_links">Links</string>
-    <string name="about_links_list"><![CDATA[\u00B7 <a href="https://github.com/openhab/openhab.android">openHAB App source code</a><br><br> \u00B7 <a href="https://github.com/openhab/openhab.android/issues">Report a problem</a><br><br> \u00B7 <a href="http://docs.openhab.org/">openHAB Documentation</a><br><br> \u00B7 <a href="https://community.openhab.org/">Community Forum</a>]]></string>
+    <string name="about_license" translatable="false">Eclipse Public License - v 1.0</string>
     <string name="about_copyright" translatable="false">© 2012 - %s openHAB Foundation</string>
     <string name="about_version_string" translatable="false">%1$s - %2$s</string>
+    <string name="about_changelog">Changelog</string>
+    <string name="about_source_code">Fork us on GitHub!</string>
+    <string name="about_issues">Report an issue</string>
+    <string name="about_docs">Documentation</string>
+    <string name="about_community_forum">Community forum</string>
+    <string name="about_foundation">openHAB Foundation</string>
+    <string name="about_community">openHAB community</string>
+    <string name="about_server">openHAB server</string>
+    <string name="about_translation">Help us to translate!</string>
```

Comment:
"Help us to translate openHAB products"?

Reply:
'Products'? I thought the link only points to the app translation? If it points to all OH translations, how about `Help us translating openHAB` or - a bit more direct - `Translate openHAB to your language`?

Reply:
It points to the profile of the openhab-bot on crowdin, which has a list with all oh projects.

Reply:
OK. Then my suggestions above are valid :-)